desktop/src/gui/controller.rs

```diff
@@ -286,3 +289,49 @@ impl GuiController {
         self.gui.open_file_advanced()
     }
 }
+
+// try to load known unicode supporting fonts to draw cjk characters in egui
+fn load_system_fonts() -> anyhow::Result<egui::FontDefinitions> {
+    let mut font_database = Database::default();
+    font_database.load_system_fonts();
+
+    let system_unicode_fonts = Query {
+        families: &[
+            Family::Name("MS UI Gothic"),     // windows
+            Family::Name("Arial Unicode MS"), // macos
+            Family::Name("Noto Sans"),        // linux
+            Family::SansSerif,
+        ],
+        ..Query::default()
+    };
```

Comment:
Is it loading only `MS UI Gothic` on Windows? If so, it will always display Japanese characters, ignoring CJK settings (locale) of the system.

Reply:
Yeah, this was pointed out by Dinnerbone as well. Do you agree with Daniel's suggested font list above?

Reply:
I'll include whatever fonts you all deem acceptable for minimum viable coverage of all character systems commonly used. is there a easy way to refer to the system locale?

Reply:
I assume that would be this:

https://github.com/ruffle-rs/ruffle/blob/fcd9935a8102717ce40d893f252da2723f343717/desktop/src/gui.rs#L21

https://github.com/ruffle-rs/ruffle/blob/fcd9935a8102717ce40d893f252da2723f343717/desktop/src/gui.rs#L80-L83

As the comment says "This should also be somewhere else so it can be supplied through UiBackend too". So right not I think it would just be by re-using that method.